src/Sulu/Bundle/ContentBundle/Resources/public/js/main.js

```diff
@@ -37,6 +39,8 @@ define([
 
             'use strict';
 
+            SeoTab.initialize(app);
```

Comment:
Isn't `app.use` the way to register extensions? I'd say extensions are also not supposed to be loaded via requirejs.